### DISCUSSION TITLE
开源之道：经验还是科学？ - 「开源之道」

### DISCUSSION THREAD

**Discussion by giscus** [OCselected/ocselected.github.io#50] [Announcements]
# 开源之道：经验还是科学？ - 「开源之道」

https://opensourceway.community/posts/the_way_of_open_source/osw-experience-and-science/

**Comment by ZoomQuiet** [comment]
#是也乎,(￣▽￣)

经验就是成功安装到人们头脑中的软件,
科学, 就是可以高效创造这种可安装到大家头脑中软件的经验;

而对开源世界的理解本身, 也是需要一套全新分析软件的,
开源之道 本身, 就是作者自我摸索这种软件构造过程的记录;
可以视之为开发笔记,
开发笔记不是开发成果,
但是, 比最终成品更加重要,

因为, 就是每个人都可以复用/参照来完成自己经验形成的...
